sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/di/UnusedSyncablePluginCodegenTrigger.kt

```diff
@@ -14,22 +14,21 @@
  * limitations under the License.
  */
 
```

Comment:
Not sure if this is a bug on github, but it seems is confusing this new class with an existing one we should not remove yet.